malformed/test_purge_hch.py

```diff
@@ -0,0 +1,135 @@
+
+from framework import tester
+from helpers import tf_cfg, deproxy
+
+__author__ = 'Tempesta Technologies, Inc.'
+__copyright__ = 'Copyright (C) 2018 Tempesta Technologies, Inc.'
+__license__ = 'GPL2'
+
+class MalformedCrlfTest(tester.TempestaTest):
+    # This is a heavy chunked test for ss_skb_chop_head_tail() function in context
+    # of rewriting PURGE method as GET, issue #1535
+    #
+    # TODO: move the test to appropriate place (/cache/?)
```

Comment:
@ad-tmpst this is the test for your https://github.com/tempesta-tech/tempesta/issues/1516 . Could you please review and comment on this?

It seems we also need to test heavily chunked response for https://github.com/tempesta-tech/tempesta/issues/1535 . @Dmitry-Gouriev did you reveal the oops from https://github.com/tempesta-tech/tempesta/issues/1535#issuecomment-988478106 by the test?

Reply:
Probably I do not understand the question.

From the other side. Additional inbestigations show, that this oops message is generated by
```
979		if (WARN_ON_ONCE(skb->len <= head + trail))
980			return -EINVAL;
```
(ss_skb.c).
So the function is guarded from more serious failures,
but, nevertheless, the TFW can not rewrite heavy chunked PURGE requests.

Reply:
I meant whether you used some test from this PR to get the oops from https://github.com/tempesta-tech/tempesta/issues/1535#issuecomment-988478106 ?

> the TFW can not rewrite heavy chunked PURGE requests.

Probably we need to work a little more on it.

What happens if Tempesta receives heavily chunked response? PURGE is a service request, produced by the site owner or CDN owner only, so it'd be OK if we just print a warning (human readable, not a call trace produced by the WARN_ON_ONCE()) and just don't do anything. I don't want to introduce a complex logic for this unprobable and non-critical code.

@ad-tmpst maybe you have something to comment on this.